### PR TITLE
Nate's basket of documentation fixes no. 3

### PR DESCRIPTION
**Pull request by nbarbettini**
Policy API reference
* Add links to the Postman collection for the Policy API 

OIDC reference
* Clean up for formatting and accuracy
* Update logout endpoint to reflect changes in OKTA-180521: expired or no Okta session results in a redirect, not a JSON error. Invalid ID tokens result in an error page redirect.

User API reference
* Minor cleanup of formatting/typos
* Clarify that links returned on the User object may vary depending on your policies

### Resolves
- [OKTA-180521](https://oktainc.atlassian.net/browse/OKTA-180521)

### CODE REVIEW

File: _source/_docs/api/resources/oidc.md [hunk 1]
```diff
@@ -508,61 +508,53 @@ Content-Type: application/json;charset=UTF-8
...
-| id_token_hint            | A valid ID token with a subject matching the current session.                                                                                   | String | TRUE     |
-| post_logout_redirect_uri | Callback location to redirect to after the logout has been performed. It must match the value preregistered in Okta during client registration. | String | FALSE    |
-| state                    | If the request contained a `state` parameter, then the same unmodified value is returned back in the response.                                  | String | FALSE    |
+| Parameter                | Description                                                                                                                              | Type   |
+|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------|:-------|
+| id_token_hint            | **Required**. A valid ID token with a subject matching the current session.                                                              | String |
```
Comment (jmelberg-okta) [inline _source/_docs/api/resources/oidc.md]: Doesn't need to be tackled with this PR, but we should probably explore finding a consistent way to callout required parameters. In this document alone, we're specifying **required** params by either a separate table column, `Required for ...`inline documentation, or `XYZ is required for ...`

File: _source/_docs/api/resources/oidc.md [hunk 2]
```diff
@@ -508,61 +508,53 @@ Content-Type: application/json;charset=UTF-8
...
 Use this operation to log out a user by removing their Okta browser session.
 
+This endpoint takes an ID token and logs the user out of Okta if the subject matches the current Okta session. A `post_logout_redirect_uri` may be specified to redirect the browser after the logout has been performed. Otherwise, the browser is redirected to the Okta login page.
+
+If no Okta session exists, this endpoint has no effect and the browser is redirected immediately to the Okta login page or the `post_logout_redirect_uri` (if specified).
+
 #### Request Parameters
 
-The following parameters can be posted as a part of the URL-encoded form values to the API.
+The following parameters can be included in the query string of the request:
```
Comment (jmelberg-okta) [inline _source/_docs/api/resources/oidc.md]: Was this incorrect before or is this new behavior?

File: _source/_docs/api/resources/oidc.md [hunk 3]
```diff
@@ -508,61 +508,53 @@ Content-Type: application/json;charset=UTF-8
...
-| state                    | If the request contained a `state` parameter, then the same unmodified value is returned back in the response.                                  | String | FALSE    |
+| Parameter                | Description                                                                                                                              | Type   |
+|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------|:-------|
+| id_token_hint            | **Required**. A valid ID token with a subject matching the current session.                                                              | String |
+| post_logout_redirect_uri | Location to redirect to after the logout has been performed. It must match the value preregistered in Okta during client registration.   | String |
+| state                    | Any `state` value is included in the redirect to the `post_logout_redirect_uri`.                                                         | String |
```
Comment (jmelberg-okta) [inline _source/_docs/api/resources/oidc.md]: I'm a bit confused by this line. I'm reading it as - We will create a `state` param and append it to the `post_logout_redirect_uri` if one is not provided.

Could we reword this to something similar to:

> An optional `state` value that is returned as a query parameter during the redirect to the `post_logout_redirect_uri`.

File: _source/_docs/api/resources/oidc.md [hunk 4]
```diff
@@ -508,61 +508,53 @@ Content-Type: application/json;charset=UTF-8
...
 #### Response Example (Success)
 
 ~~~http
-HTTP 302 FOUND
+HTTP 302 Found
+Location: https://...
 ~~~
 
-Followed by either a redirect to your org's login URL, or (if applicable) the specified logout redirect URI.
+This will redirect the browser to either the Okta login page or the specified logout redirect URI.
```
Comment (jmelberg-okta) [inline _source/_docs/api/resources/oidc.md]: redirect the browser -> redirect the user

File: _source/_docs/api/resources/oidc.md [hunk 5]
```diff
@@ -508,61 +508,53 @@ Content-Type: application/json;charset=UTF-8
...
-    "errorCode": "E0000005",
-    "errorSummary": "Invalid session",
-    "errorLink": "E0000005",
-    "errorId": "oae4uSVaLVbRTOoRB_EsrXMWw",
-    "errorCauses": []
-}
-~~~
+If the Okta session has expired (or does not exist), a logout request will simply redirect to the Okta login page or the `post_logout_redirect_uri` (if specified).
+
+If the ID token passed via `id_token_hint` is invalid or expired, the browser will be redirected to an error page.
```
Comment (jmelberg-okta) [inline _source/_docs/api/resources/oidc.md]: browser will be redirected -> browser will redirect

File: _source/_docs/api/resources/oidc.md (hunk 1 above)
Comment (nbarbettini) [reply _source/_docs/api/resources/oidc.md]: I agree, it's something that I've discussed with @jakubvul a few times. In this table, it seemed particularly unnecessary to have a Required column where every entry was False except for one.

File: _source/_docs/api/resources/oidc.md (hunk 2 above)
Comment (nbarbettini) [reply _source/_docs/api/resources/oidc.md]: Incorrect before. This has always been a redirect (GET) request with query parameters, not a form body.

File: _source/_docs/api/resources/oidc.md (hunk 3 above)
Comment (robertjd) [reply _source/_docs/api/resources/oidc.md]: +1 that reads better

Comment (nbarbettini) [reply _source/_docs/api/resources/oidc.md]: 👍

**Comment by nbarbettini** [comment]
Rebased to clean up commit log
